calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| `local_map_num_keyframes`                | `int`                 | `15`           | The number of keyframes stored in the local map.                                                                                                                                              |
+| `dense_pointcloud_num_keyframes`         | `int`                 | `10`           | Dense pointcloud for calibration is generated by integrating the pointcloud in the range of [keyframe_id - `dense_pointcloud_num_keyframes`, keyframe_id + `dense_pointcloud_num_keyframes`]. |
+| `mapping_min_range`                      | `double`              | `0.5`          | The minimum range in meters of each lidar pointcloud for mapping.                                                                                                                             |
+| `mapping_max_range`                      | `double`              | `60.0`         | The maximum range in meters of each lidar pointcloud for mapping.                                                                                                                             |
+| `min_mapping_pointcloud_size`            | `int`                 | `10000`        | The minimum size of the pointcloud required for mapping.                                                                                                                                      |
+| `min_calibration_pointcloud_size`        | `int`                 | `500`          | The minimum size of the pointcloud that is necessary for estimating transformation.                                                                                                           |
+| `mapping_lost_timeout`                   | `double`              | `1.0`          | Sensor's timeout in seconds to consider the mapping process is failed.                                                                                                                        |
+
+### Mapping Parameters
+
+| Name                                 | Type     | Default Value | Description                                                                                                                        |
+| ------------------------------------ | -------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `mapper_resolution`                  | `double` | `5.0`         | Resolution for `pclomp::NormalDistributionsTransform` algorithm.                                                                   |
+| `mapper_step_size`                   | `double` | `0.1`         | Step size for `pclomp::NormalDistributionsTransform` algorithm.                                                                    |
+| `mapper_max_iterations`              | `int`    | `35`          | The maximum number of iterations for `pclomp::NormalDistributionsTransform` and `pcl::GeneralizedIterativeClosestPoint` algorithm. |
```

Comment:
Throughout the table:
pclomp::NormalDistributionsTransform
is a proper noun

Reply:
For example:

`Leaf size in meters for pcl::VoxelGrid to voxelize the mapping pointcloud.` is Ok
`for pclomp::NormalDistributionsTransform algorithm` is not